packages/commercetools/theme/components/SfRadio.vue

```diff
@@ -0,0 +1,222 @@
+<template>
```

Comment:
Lets move adress picker + this into `components/temp` to split normal components and ones that should be removed

Reply:
Ok, moved each 3